docker-entrypoint.sh

```diff
@@ -49,6 +49,9 @@ then
   echo "Alert Database user secret set; using value from secret."
   alertDatabaseUser=$(cat $dockerSecretDir/ALERT_DB_USERNAME)
   export ALERT_DB_USERNAME=$alertDatabaseUser;
+
+  alertDatabaseAdminUser=$alertDatabaseUser;
```

Comment:
Is this set to the appropriate value? Shouldn't it use the Admin user?

Reply:
The entrypoint has to support the exernal database cases, the helm case which requires a separate password for the admin user due to the database image used,  and the case where we are using the internal database with docker swarm.  So for the internal database in docker swarm the database user and admin user are the same.  For helm with the internal database the admin user name is postgres but the password can be set.  For the external database case you have to set both the user name and password.  

The placement of where the environment variables are set is important to handle those cases.